Snippets/Snippets_6/Pipeline/SampleBehavior.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Snippets6.Pipeline
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus;
+    using NServiceBus.Pipeline;
+
+    #region SamplePipelineBehavior
+
+    public class SampleBehavior : Behavior<IncomingLogicalMessageContext>
```

Comment:
that's a reason why I was against public context implementations ;) Good catch, will fix.